udata/core/reports/api.py

```diff
@@ -37,6 +37,14 @@ def post(self):
         return report, 201
 
 
+@ns.route("/<report:report>/", endpoint="report")
+class DataserviceAPI(API):
+    @api.doc("get_report")
+    @api.marshal_with(Report.__read_fields__)
+    def get(self, report):
+        return report
```

Comment:
Admin only? (maybe I didn't do it for list endpoint :-) )

Reply:
Indeed, we were talking about it with @Pierlou yesterday! I've made these routes admin only in https://github.com/opendatateam/udata/pull/3115/commits/8196473956f37ee98664ba524ad005e4c0d66cd7